main.go

```diff
@@ -38,22 +39,23 @@ func main() {
 
 	err = telemetry.Track(context.Background(), shared.InteractionTypeCiExec, func(ctx context.Context, event *shared.CliEvent) error {
```

Comment:
@ThomasRooney 

Do you think we should omit the action from this wrapped InteractionTypeCiExec event. It's basically going to end up being a duplicate event except it will be the wrong interaction type.

Reply:
yep; it sets up a common UUID to tie everything together under `execution_id`